pep-0692.rst

```diff
@@ -463,13 +484,15 @@ previous example::
     >>> def foo(**kwargs: **Movie): ...
     ...
     >>> foo.__annotations__
-    {'kwargs': **Movie}
+    {'kwargs': Unpack[Movie]}
 
-The double asterisk syntax should call the ``__unpack__`` special method on
-the object it was used on. This means that ``def foo(**kwargs: **T): ...`` is
-equivalent to ``def foo(**kwargs: T.__unpack__()): ...``. In addition,
-``**Movie`` in the example above is the ``repr`` of the object that
-``__unpack__()`` returns.
+To accomplish this, we propose a new dunder called ``__typing_kwargs_unpack__``.
+The double asterisk syntax should result in a call to the
+``__typing_kwargs_unpack__`` special method on an object it was used on.
+This means that at runtime, ``def foo(**kwargs: **T): ...`` is equivalent to
+``def foo(**kwargs: T.__typing_kwargs_unpack__()): ...``.
```

Comment:
Is this meant literally (call the method on the type instance), or as a shortcut for normal Python calling of magic methods which is `type(T).__typing_kwargs_unpack(T)` ?

Reply:
I meant this literally - wasn't aware that python calls magic methods as you've described. Do you think it would make sense to change this description? To me it seems that for simplicity it could be left as is.

Reply:
The audience for a PEP is implementors, so being precise and unambiguous is usually better.

Reference for special method lookup: https://docs.python.org/3/reference/datamodel.html#special-method-names

Reply:
Thanks